src/explorer.api/Models/NumericOverview.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace Explorer
```

Comment:
I am not sure that this class and its base class (ColumnExplorer) belong under the Models folder.

Reply:
Ok, I'm not sure I understand the asp.net definitions of models vs controllers vs everything else. Better in the top-level folder then? 

What kinds of things should go in the Models folder? Only objects that are actually sent to / received by the api?

Reply:
I am not sure also, I didn't use ASP.NET core before. But I think that the models should contain just the objects sent to/received by the API, as you said; that is simple classes that usually contain just fields and don't have too much additional functionality.

Reply:
That makes sense. I've reorganised the file structure in the latest push.